source/material_model/rheology/peierls_creep.cc

```diff
@@ -340,13 +320,24 @@ namespace aspect
         * deriv = edot_ii / stress * (s + n)
         */
         const PeierlsCreepParameters p = creep_parameters;
-    if (stress < p.stress_c)
+    if (stress < p.stress_coff)
       {
-        const double b = (p.activation_energy + pressure*p.activation_volume)/(constants::gas_constant * temperature);
-        const double arrhenius = std::exp(-b*p.d_ref);
+        const double T_coff = p.activation_energy/constants::gas_constant;
+        const double P_coff = 0.;
+        const double b_coff = (p.activation_energy + P_coff*p.activation_volume)/(constants::gas_constant*T_coff);
+        const double c_coff = std::pow(p.stress_coff/p.peierls_stress, p.glide_parameter_p);
+        const double d_coff = std::pow(1. - c_coff, p.glide_parameter_q);
+        const double s_coff = b_coff*p.glide_parameter_p*p.glide_parameter_q*c_coff*d_coff/(1. - c_coff);
+        const double arrhenius_coff = std::exp(-b_coff * d_coff);
```

Comment:
```suggestion
        const double arrhenius_coff = std::exp(-d_coff);
```
(and change names from coff -> cutoff)